src/azure-cli/azure/cli/command_modules/role/_help.py

```diff
@@ -397,21 +402,51 @@
 
 helps['ad group'] = """
 type: group
-short-summary: Manage Azure Active Directory groups.
+short-summary: Manage groups.
 """
 
 helps['ad group create'] = """
 type: command
-short-summary: Create a group in the directory.
+short-summary: Create a group.
 examples:
   - name: Create a group in the directory. (autogenerated)
     text: az ad group create --display-name MyDisplay --mail-nickname MyDisplay
     crafted: true
 """
 
+helps['ad group show'] = """
+type: command
+short-summary: Get the details of a group.
+"""
+
+helps['ad group delete'] = """
+type: command
+short-summary: Delete a group.
+"""
+
+helps['ad group get-member-groups'] = """
+type: command
+short-summary: Get a collection of object IDs of groups of which the specified group is a member.
+"""
+
 helps['ad group member'] = """
 type: group
-short-summary: Manage Azure Active Directory group members.
+short-summary: Manage group members.
```

Comment:
```suggestion
short-summary: Manage Microsoft Entra ID group members.
```

Reply:
According to https://learn.microsoft.com/en-us/entra/fundamentals/new-name and https://learn.microsoft.com/en-us/entra/fundamentals/how-to-rename-azure-ad, the new name should be `Microsoft Entra group`.

![image](https://github.com/Azure/azure-cli/assets/4003950/8a04e41b-09f0-4c7e-b783-fa4ccd5550f3)

```
    @{ Key = 'Azure Active Directory group'; Value = 'Microsoft Entra group' },
```